spec/index.adoc

```diff
@@ -1471,15 +1473,15 @@ For each canister, the IC keeps track of “certified data”, a canister-define
 +
 The canister can update the certified data with this call. The passed data must be no larger than 32 bytes. This can be used any number of times.
 
-When executing a query method via a query call (i.e. in non-replicated state), the canister can fetch a certificate that authenticates to third parties the value last set via `ic0.certified_data_set`.
+When executing a query or composite query method via a query call (i.e. in non-replicated state), the canister can fetch a certificate that authenticates to third parties the value last set via `ic0.certified_data_set`.
 
 * `+ic0.data_certificate_present : () -> i32+`
 +
 returns `1` if a certificate is present, and `0` otherwise.
 +
-This will return `1` when called from a query method when invoked via a query call.
+This will return `1` when called from a query or composite query method via a query call.
 +
-This will return `0` if the query method is executed within replicated execution (e.g. when invoked via an update call or inter-canister call).
+This will return `0` for update methods and if a query method is executed within replicated execution (e.g. when invoked via an update call or inter-canister call).
```

Comment:
```suggestion
This will return `0` for update methods, if a query method is executed within replicated execution (e.g. when invoked via an update call or inter-canister call) and from within composite query calls for all but the initial method invoked by the user.
```